pkg/antctl/raw/traceflow/command_test.go

```diff
@@ -153,3 +197,360 @@ func TestParseFlow(t *testing.T) {
 		}
 	}
 }
+
+func TestRunE(t *testing.T) {
+	tcs := []struct {
+		name        string
+		src         string
+		dst         string
+		outputType  string
+		liveTraffic string
+		droppedOnly string
+		expected    string
+	}{
+		{
+			name:     "no source",
+			expected: "Please provide source",
+		},
+		{
+			name:     "no destination",
+			src:      srcPod,
+			expected: "Please provide destination",
+		},
+		{
+			name:        "live traffic",
+			liveTraffic: "1",
+			expected:    "One of source and destination must be a Pod",
+		},
+		{
+			name:        "dropped-only",
+			src:         srcPod,
+			dst:         ipv4,
+			droppedOnly: "1",
+			expected:    "--dropped-only works only with live-traffic Traceflow",
+		},
+		{
+			name:       "dummy-traceflow-pod-to-pod",
+			src:        srcPod,
+			dst:        dstPod,
+			outputType: "yaml",
+			expected: `phase: Succeeded
+source: default/pod-1
+destination: default/pod-2
+`,
+		},
+		{
+			name:       "dummy-traceflow-pod-to-service",
+			src:        srcPod,
+			dst:        "default/service",
+			outputType: "yaml",
+			expected: `phase: Succeeded
+source: default/pod-1
+destination: default/service
+`,
+		},
+		{
+			name:       "dummy-traceflow-pod-to-ipv4",
+			src:        srcPod,
+			dst:        ipv4,
+			outputType: "json",
+			expected: `
+  "phase": "Succeeded",
+  "source": "default/pod-1",
+  "destination": "192.168.10.10"
```

Comment:
I think json format should contain `{}`? It looks strange without it.

Reply:
I cannot because it contains name before phase in actual output and i cannot predict name and before name there is `{`, i can only put `}` at ending because there are no fields after destination but only adding this `}` at end and not in beginning would look strange. You can also I used assert.Contains and not assert.Equal

Reply:
Got it.